client/engine/store/mockstore.go

```diff
@@ -206,6 +206,27 @@ func (ms *MockStore) GetTwoPartyLedger(firstParty types.Address, secondParty typ
 	return ledger, ok
 }
 
+// GetConsensusChannel returns a ConsensusChannel between the calling client and
+// the supplied counterparty, if such channel exists
```

Comment:
This godoc needs to be updated for `GetConsensusChannelById`

Reply:
5f374aa95af8714380d83631430525a709793867 (in Part 1)